IdentityCore/src/util/NSString+MSIDExtensions.m

```diff
@@ -259,14 +259,21 @@ - (BOOL)msidIsEquivalentWithAnyAlias:(NSArray<NSString *> *)aliases
     {
         return NO;
     }
-
+    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"Check if credential environment %@ is included in the list of valid environment aliases.",self);
+    for (NSString *alias in aliases)
+    {
+        MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, nil, @"environment alias found: %@",alias);
+    }
+    
     for (NSString *alias in aliases)
     {
         if ([self caseInsensitiveCompare:alias] == NSOrderedSame)
         {
+            MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"credential environment found in the list of valid environment aliases.");
```

Comment:
This is a bit too verbose for Info level since we perform this operation quite often

Reply:
Got it. changing it to verbose then.